lib/http/HttpServer.ts

```diff
@@ -41,7 +41,7 @@ class HttpServer {
         }
       } catch (err) {
         const msg = `processing request to ${req.url} failed`;
-        this.logger.error(`${msg}: ${err}`);
```

Comment:
This could be

`this.logger.error(msg, err);`

Because the `error` log method already accepts a second parameter for the error and will automatically JSON.stringify it if it's an object.

Reply:
Thanks - good catch. Updated @sangaman.